.github/workflows/ci-docs.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       # We only use a non-zero build # when making multiple manual builds in one day.
```

Comment:
> Update the version number to v11.2 to pull in #7156 .

In the PR description, would be nice to include a one-liner for what changed that prompted the new release: the trace version had to be bumped from an earlier PR. That it is important to determine analysis tool compatibility with trace format.

Reply:
Done.